config/routes.rb

```diff
@@ -452,6 +453,15 @@
       resources :reference_timelines, path: 'timelines', except: [:new, :edit, :show] do
         resources :reference_times, path: 'times', only: [:create, :update, :destroy]
       end
+
+      scope :monitoring do
```

Comment:
This is actually a controller route. I don't think there is a notion of "routes" in Action Cable, because it's a WebSocket. Connect to an endpoint, subscribe to whatever channel you need, and all these are identified by `params`.

There is, however, a way to specify the Action Cable route in `routes.rb` by mounting it as a Rack application.
```ruby
mount ActionCable.server, at: '/my_socket'
```
but this this essentially the same as configuring `config.action_cable.url` in `application.rb`. Notice how I wrote _**the** Action Cable route_ because Rails doesn't yet support multiple Action Cable servers in one app.

I didn't set all these because the default is `'/cable'` and I'm using it as is.

Reply:
Oh, crap. I was seeing the wrong thing. I'm going to delete this scope. This was added back when we weren't doing WebSocket. Sorry for the confusion.

The only thing change to the `routes.rb` file is line 203.